src/content/guides/farmlands-portal.mdx

```diff
@@ -59,69 +59,67 @@ The Centrapay Business Portal provides a real-time authorisation of a Farmlands
 
 ### Create a Farmlands Authorisation
 
-1. Select Create Authorisation Request from the main dashboard.
+1. Select **Create Authorisation Request** from the main dashboard.
 
     ![Create a Farmlands authorisation request on the Centrapay business portal](/farmlands-payment-create-authorization-request.png)
 
-2. Enter the total amount of the sale (including GST)
+2. **Enter the total amount** of the sale (including GST)
 
     > Make sure the correct merchant is selected
 
     ![Enter payment amount on the Centrapay business portal](/farmlands-payment-enter-amount.png)
 
-3. Scan the barcode on the rear of the Farmlands Card by using the camera function, scanner (if available). Alternatively, manually enter the 9-digit Card number located on the front of the Card (bottom left-hand corner or below the barcode on the rear of the Card).
+3. **Scan the barcode** on the rear of the Farmlands Card by using the camera function or scanner (if available). Alternatively, manually enter the 9-digit Card number located on the front of the Card (bottom left-hand corner or below the barcode on the rear of the Card).
 
-    > Enter the 9-digit card number, not the 16-digit card number or 9-digit account number.
+    > The 9-digit Card number displayed on the Card must be entered, not the 16-digit card number or 9-digit account number.
```

Comment:
Using "The 9-digit number displayed on the Card must be entered" would be more concise but if this is what Farmlands have suggested then just keep it as is.